### DISCUSSION TITLE
Active Colors Not Showing for Breadcrumbs

### DISCUSSION THREAD

**Discussion by Cassianky** [quasarframework/quasar#13824] [General - Components / Directives / etc] [answered]
![Screenshot (160)](https://user-images.githubusercontent.com/77654564/176578343-4ce1bdaa-cf66-4a34-825d-07086945d389.png)

This is a comparison of the v1 and v2 quasar documentation regarding breadcrumbs. Even though the sample code is identical, but the active colours for breadcrumb is not showing for v2 quasar documentation. I saw a similar thread that was resolved with the update offered in quasar v (link: https://github.com/quasarframework/quasar/issues/11959). But I could replicate the error in quasar v2.7.1 as well. 

I'm a beginner in quasar so I'm not sure if my usage of breadcrumbs like in the documentation/link is correct. Thanks in advance.

**Comment by pdanpdan** [comment]
Thank you for reporting

https://github.com/quasarframework/quasar/pull/13825

**Comment by rstoenescu** [comment, marked as answer]
Fix will be available in Quasar v2.7.5
